SCClassLibrary/Common/Control/Buffer.sc

```diff
@@ -585,14 +585,18 @@ Buffer {
 		^[\b_close, bufnum, completionMessage.value(this) ]
 	}
 
-	query {
+	query { |action|
```

Comment:
ah, actually, could you be so kind and also document the new argument in the `Buffer` helpfile?

Reply:
Done! Please suggest any improvements.